acceptance/acceptance_test.go

```diff
@@ -431,13 +431,32 @@ func testWithoutSpecificBuilderRequirement(
 		})
 
 		when("default builder is set", func() {
-			it("outputs information", func() {
+			it("redacts default builder", func() {
 				pack.RunSuccessfully("set-default-builder", "paketobuildpacks/builder:base")
 
 				output := pack.RunSuccessfully("report")
 
 				version := pack.Version()
 
+				expectedOutput := pack.FixtureManager().TemplateFixture(
+					"report_output.txt",
+					map[string]interface{}{
+						"DefaultBuilder": "[REDACTED]",
+						"Version":        version,
+						"OS":             runtime.GOOS,
+						"Arch":           runtime.GOARCH,
+					},
+				)
+				assert.Equal(output, expectedOutput)
+			})
+
+			it("explicit mode doesn't redact", func() {
+				pack.RunSuccessfully("set-default-builder", "paketobuildpacks/builder:base")
+
+				output := pack.RunSuccessfully("report", "-e")
```

Comment:
Minor: This would be slightly clearer if the full name flag was used.

```suggestion
				output := pack.RunSuccessfully("report", "--explicit")
```